TEST_DEV_GUIDE.md

```diff
@@ -123,5 +122,4 @@ See the [README](../aries-agent-test-harness/aries-backchannels/README.md) in th
 
 ## Diving Deeper
 - [Accessing Connection ID in Test Code](ACCESS-CONNECTION-IDS.md)
-- [Configuring Tests with Credential Types and Proofs](CONFIGURE-CRED-TYPES.md)
-- [Debugging a Backchannel Running Inside a Docker Contaiener](DEBUGGING.md)
```

Comment:
Same?

Reply:
hmmm, that is strange. I have no idea what happened here.

Reply:
File properly merged in new commit.